Dockerfile.Windows

```diff
@@ -1,9 +1,17 @@
 ARG BASE_IMAGE
 ARG BASE_IMAGE_TAG
+FROM --platform=$BUILDPLATFORM golang:1.13.3 AS builder
+
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+WORKDIR /code
+ADD . /code/
+
+RUN cd /code/ && GOARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') GCE_PD_CSI_STAGING_VERSION=dev make gce-pd-driver-windows
```

Comment:
Should GCE_PD_CSI_STAGING_VERSION be hardcoded?

Reply:
fixed.